Snowflake.Data.Tests/IntegrationTests/SFPutGetTest.cs

```diff
@@ -342,7 +346,7 @@ public void TestPutDirectoryMixedWildcard()
         [Test]
         [Parallelizable(ParallelScope.All)]
         public void TestPutGetCommand(
-            [Values("gzip", "bzip2", "brotli", "deflate", "raw_deflate", "zstd")] string compressionType, 
+            [Values("none", "gzip", "bzip2", "brotli", "deflate", "raw_deflate", "zstd")] string compressionType,
             [Values] StageType stageType,
             [Values("", "/TEST_PATH", "/DEEP/TEST_PATH")] string stagePath,
             [Values] bool autoCompress)
```

Comment:
what autoCompress is referring to, maybe we can rename to some more meaningful name

Reply:
I renamed compressionType to sourceFileCompressionType and further t_compressionType to t_sourceCompressionType.
AutoCompress is an option to use in the connection string. I think it's name is good since it is a part of the API.